evergreen.yml

```diff
@@ -41,6 +41,44 @@ functions:
       params:
         file: mongo-tableau-connector/expansions.yml
 
+  "setup python virtual environment":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongo-tableau-connector
+        script: |
+          ${prepare_shell}
+
+          python3 -m venv .venv
+
+
+  "install connector plugin sdk":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongo-tableau-connector
+        script: |
+          ${prepare_shell}
+
+          # Clone connector-plugin-sdk and install packaging module
+          git clone https://github.com/tableau/connector-plugin-sdk.git
+          source ./.venv/bin/activate
+          ( cd connector-plugin-sdk/connector-packager && python3 setup.py install )
+
+  "validate connector XML format":
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-tableau-connector
+        script: |
+          ${prepare_shell}
+          source ./.venv/bin/activate
+          # Running packager validation step
+          ( cd connector-plugin-sdk/connector-packager && python3 -m connector_packager.package \
+          --validate-only --verbose ../../connector/ --log ../../logs 2>&1 | tee /dev/stdout \
+          | grep "Validation succeeded." >/dev/null)
```

Comment:
Added these grep commands as the python call was exiting with `0` on success and failure.